evals/solvers/providers/fixie/local_gpu_solver.py

```diff
@@ -101,6 +107,27 @@ def __del__(self):
             self.executor.shutdown()
 
 
+class DataCollatorForSeq2SeqWithAudio(transformers.DataCollatorForSeq2Seq):
```

Comment:
I would've loved not to have specialized Ultravox code here, but I needed the collator and there wasn't a better way to access it.